tests/pom.xml

```diff
@@ -104,100 +104,6 @@
             <artifactId>spark-avro_${scala.binary.version}</artifactId>
         </dependency>
     </dependencies>
-
```

Comment:
Thank you very much for cleaning this up. I suggest to create a dedicated PR for this to keep this PR focused on DBR 13.3